eng/build.sh

```diff
@@ -80,7 +80,8 @@ while [[ $# > 0 ]]; do
       shift 2
       ;;
      -configuration|-c)
-      arguments="$arguments /p:ConfigurationGroup=$2 -configuration $2"
+      val="$(tr '[:lower:]' '[:upper:]' <<< ${2:0:1})${2:1}"
```

Comment:
out of curiosity what does: ` <<< ${2:0:1})${2:1}` do?

Reply:
Uppercase the first character (0-1) and append the remaining ones (1-n).